website/config/algolia/crawler-settings.js

```diff
@@ -13,23 +13,29 @@ new Crawler({
       recordExtractor: ({ $, helpers }) => {
         // priority order: deepest active sub list header -> navbar active item -> 'Documentation'
         const lvl0 =
-          $(
-            ".menu__link.menu__link--sublist.menu__link--active, .navbar__item.navbar__link--active"
-          )
-            .last()
-            .text() || "Documentation";
+          $(".navbar__item.navbar__link--active").last().text() ||
+          "Documentation";
+        const isApi = $(".menu__link.menu__link--active")
+          .toArray()
+          .some((element) => $(element).text() === "API")
+          ? "yes"
+          : "no";
         return helpers.docsearch({
           recordProps: {
             lvl0: {
               selectors: "",
               defaultValue: lvl0,
             },
-            lvl1: "header h1",
+            lvl1: "article h1",
```

Comment:
For some reason the default selector here was buggy, and only worked if title was defined in frontmatter, not when it was just an h1 tag, as is the case for most of our docs